### ISSUE TITLE
RBE by default?

### ISSUE THREAD

**Issue by mkgold** [cuny-academic-commons/commons-in-a-box#112]
Should RBE be on by default on new CBOX installations with forums activated?

If not, is it clear enough to users that it exists and that they can turn it on if they want it?

**Comment by boonebgorges** [comment]
I'd say we should not enable it by default.

RBE has two modes, IMAP and Mandrill. Neither one works without a fair amount of external configuration (in one case, the creation of an email inbox and the enabling of a non-default PHP module; in the other, signing up for a non-free, external account and making DNS changes). It seems like a bad experience to enable something by default that will not work, and perhaps cannot work, for a majority of users.

There may be a better way to advertise to users that RBE, and other "a la carte" plugins, are available. I'm not sure what that might be, though. Ideas welcome. cc @samradd

**Comment by r-a-y** [comment]
> Should RBE be on by default on new CBOX installations with forums activated?

I agree with Boone.  RBE is not a plug-and-play plugin.  For that reason, I would say no.

> If not, is it clear enough to users that it exists and that they can turn it on if they want it?

RBE is listed in the "A la carte" plugins section on the CBOX plugins page.  So people are able to activate it if they wanted to.

I echo what Boone mentioned that the "A la carte" plugins could possibly be better advertised.

**Comment by mkgold** [comment]
Okay. Can we ask Sam about better advertising a la carte plugins?

**Comment by samradd** [comment]
Could we have boxes on the top of the dashboard? A la this (see what I did there?): 
![screen shot 2016-03-14 at 1 53 19 pm](https://cloud.githubusercontent.com/assets/7431171/13754175/6091a5dc-e9ec-11e5-8b2c-54c35789f521.png)
These seem to be pretty standard in the purchased themes I've been using lately. And as long as closing them gets rid of them forever, I think they're relatively non-intrusive.

**Comment by r-a-y** [comment]
We currently have two _A la carte_ plugins: Reply By Email and External Group Blogs.  With this section likely to grow in the future, IMO, a notice isn't really suitable.

Maybe we can advertise this better on the CBOX admin dashboard page.

See the first screenshot on this page (it's a little outdated, but hopefully you get a sense of it):
http://commonsinabox.org/documentation/cbox-dashboard
